ORK1Kit/ORK1Kit/Common/ORK1AnswerFormat.m

```diff
@@ -2493,6 +2509,7 @@ - (ORK1AnswerFormat *)impliedAnswerFormat {
         _impliedAnswerFormat.spellCheckingType = UITextSpellCheckingTypeNo;
         _impliedAnswerFormat.autocapitalizationType = UITextAutocapitalizationTypeNone;
         _impliedAnswerFormat.autocorrectionType = UITextAutocorrectionTypeNo;
+        _impliedAnswerFormat.textContentType = UITextContentTypeEmailAddress;
```

Comment:
Shouldn't this have the same `usernameField ? UITextContentTypeUsername : UITextContentTypeEmailAddress` logic that you're using below?